code_examples/sdk_examples/src/core_features/did/09_did_signature.ts

```diff
@@ -1,20 +1,32 @@
 import * as Kilt from '@kiltprotocol/sdk-js'
+import { u8aToHex } from '@polkadot/util'
+
+type KeyLookup = (parameter: {
+  didUri: Kilt.DidUri
+  keyRelationship: Kilt.VerificationKeyRelationship
+}) => Promise<{
+  key: Kilt.KiltKeyringPair
+  keyType: Kilt.VerificationKeyType
+  keyUri: Kilt.DidResourceUri
+}>
```

Comment:
This is a big leap. Where do we explain the concept of key storage. Most places we generate keys on the mnemonic. This has taken a big step in assuming that they store their key. Also, this would need explaining and in the text I do not see this.

Reply:
Where to get the keys from is a new topic. This is far to complex and depends on quite a lot of things. We can't explain this in this section. That's the reason for having this lookup method. They need to look it up themself.